misk-admin/src/main/kotlin/misk/web/dashboard/DashboardTabModule.kt

```diff
@@ -0,0 +1,159 @@
+package misk.web.dashboard
+
+import misk.inject.KAbstractModule
+import misk.web.WebActionModule
+import misk.web.v2.DashboardHotwireTabAction
+import misk.web.v2.DashboardIFrameTabAction
+import misk.web.v2.DashboardPageBuilder.Companion.BETA_PREFIX
+import misk.web.v2.MiskWebLoaderAction
+
+/** Handles complete installation of a new DashboardTab for use in AdminDashboard or another. */
+class DashboardTabModule(
```

Comment:
This module replaces the previous two calls to `multibind<DashboardTab>().toProvider(DashboardTabProvider(...)` and `install(WebTabResourceModule(...)` which previously were used to install a Misk-Web tab.

This new module encapsulates both of those making it easier to do any migrations going forward and to also support Misk-Web, Hotwire, IFrame tab nuances in a single place.